flexmeasures/data/schemas/reporting/pandas_reporter.py

```diff
@@ -43,16 +47,44 @@ def validate_method_call(self, data, **kwargs):
             )
 
 
-class PandasReporterConfigSchema(ReporterConfigSchema):
+class BeliefsSearchConfigSchema(Schema):
+    """
+    This schema implements the required fields to perform a TimedBeliefs search
+    using the method flexmeasures.data.models.time_series:Sensor.search_beliefs
+    """
+
+    sensor = SensorIdField(required=True)
+
+    event_starts_after = AwareDateTimeField()
+    event_ends_before = AwareDateTimeField()
+
+    belief_time = AwareDateTimeField()
+
+    horizons_at_least = DurationField()
+    horizons_at_most = DurationField()
+
+    source = DataSourceIdField()
+
+    source_types = fields.List(fields.Str())
+    exclude_source_types = fields.List(fields.Str())
+    most_recent_beliefs_only = fields.Boolean()
+    most_recent_events_only = fields.Boolean()
+
+    one_deterministic_belief_per_event = fields.Boolean()
+    one_deterministic_belief_per_event_per_source = fields.Boolean()
+    resolution = DurationField()
+    sum_multiple = fields.Boolean()
+
+
+class PandasReporterConfigSchema(Schema):
     """
     This schema lists fields that can be used to describe sensors in the optimised portfolio
 
     Example:
 
     {
-        "input_sensors" : [
-            {"sensor" : 1, "alias" : "df1"}
-        ],
+        "sensor" : 1,
```

Comment:
Isn't this the output sensor? While the df variable names suggest ID=1 is used for an input sensor.

Reply:
True! This is outdated though as we are moving the sensor to the `input`.